src/manifest/MsQuicEtw.man

```diff
@@ -551,6 +557,12 @@
                 name="DatapathFeatures"
                 />
           </template>
+          <template tid="tid_LIBRARY_INIT_V1">
```

Comment:
Let's call it V2, since it's the second one. And please update everything else accordingly.